pkgs/tools/admin/netbox2netshot/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "netbox2netshot";
+  version = "0.1.12";
+
+  src = fetchFromGitHub {
+    owner = "scaleway";
+    repo = pname;
```

Comment:
```suggestion
    repo = "netbox2netshot";
```

Reply:
what is the reason for changing this?

Reply:
see this thread: https://github.com/NixOS/nixpkgs/pull/236604#discussion_r1222739691

Reply:
Changed it. To be honest I don't think that I should have had to change this since we don't have a

Reply:
Changed it. To be honest I don't think that I should have had to change this since we don't have anything official and there are more then enough examples in nixpkgs using this pattern, but I'm fine either ways, basically just want a decision.

Reply:
I already asked for such things, and there are none. I think this is something we should work during holidays. We should settle down in a document all the rules a contribution should follow.
It's not going to be easy, but we can try.